### ISSUE TITLE
Error: utils.owner() requires a string.

### ISSUE THREAD

**Issue by airtonix** [assemble/verb#42]
Quoting the readme:
```
Add a .verb.md. template to your project and run verb from the command line to generate the project's readme using data from package.json.
```

Did that: 

1. copied the example `.verb.md`
2. `$ npm install -g verb`
3. `$ verb`
4. got the error 

```
Error: utils.owner() requires a string.
    at Object.owner (/home/zenobius/.nvm/v0.10.35/lib/node_modules/verb-cli/node_modules/verb/lib/shared/utils.js:323:11)
    at module.exports (/home/zenobius/.nvm/v0.10.35/lib/node_modules/verb-cli/node_modules/verb/lib/transforms/username.js:18:20)
    at Template.transform (/home/zenobius/.nvm/v0.10.35/lib/node_modules/verb-cli/node_modules/verb/node_modules/template/index.js:328:8)
    at Verb._defaultTransforms (/home/zenobius/.nvm/v0.10.35/lib/node_modules/verb-cli/node_modules/verb/index.js:212:8)
    at Verb._initialize (/home/zenobius/.nvm/v0.10.35/lib/node_modules/verb-cli/node_modules/verb/index.js:80:8)
    at new module.exports.Template.extend.constructor (/home/zenobius/.nvm/v0.10.35/lib/node_modules/verb-cli/node_modules/verb/index.js:54:10)
    at Object.<anonymous> (/home/zenobius/.nvm/v0.10.35/lib/node_modules/verb-cli/node_modules/verb/index.js:685:18)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
```

package.json is here: https://github.com/airtonix/template-nunjucks/blob/master/package.json

**Comment by jonschlinkert** [comment]
thanks for the package.json, really useful!

it's the `owner` transform that figures out the `owner` part of a repo url. I temporarily have errors thrown to make it easier to debug. I'll push up a fix

**Comment by jonschlinkert** [comment]
nice! template-nunjucks looks great!

**Comment by airtonix** [comment]
cheers, btw tracked the bug down to : 

https://github.com/assemble/verb/blob/master/lib/transforms/username.js#L16

I think somewhere else you construct an object of determined values based off the package.json.

**Comment by jonschlinkert** [comment]
> I think somewhere else you construct an object of determined values based off the package.json (and values from .git/config ?).

that's right. I have a bunch of fixes to push up once I get unit tests implemented. The hard part is testing scenarios for how other people initialize their own projects. e.g. with an without `.git/config`, with `package.json` fields as strings or objects or arrays, etc.

**Comment by mightyiam** [comment]
:+1:

**Comment by jonschlinkert** [comment]
quick update, I've completely refactored verb - it's almost done. I'm now working on storing global config values for a user to make the user experience more seamless. I'll update as soon as its done

**Comment by mightyiam** [comment]
I miss verb.

**Comment by nknapp** [comment]
Which version of verb introduced the bug? Are there any workarounds? [I'm using this package.json](https://github.com/nknapp/bootprint-base/blob/master/package.json)

**Comment by jonschlinkert** [comment]
Sorry it took so long!  This should be resolved by the 0.7.0 release. 0.6.0 of verb-cli has also been released. 

Please see the [0.7.0 notes](https://github.com/assemble/verb/issues/44), there is a lot more in the release than I had time to write about, I'll add more asap!
